gtfs-validator/main/src/main/java/org/mobilitydata/gtfsvalidator/notice/CalendarHasNoActiveDaysNotice.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.notice;
+
+import com.google.common.collect.ImmutableMap;
+import org.mobilitydata.gtfsvalidator.table.GtfsCalendarService;
+
+public class CalendarHasNoActiveDaysNotice extends Notice {
+  public CalendarHasNoActiveDaysNotice(String serviceId, GtfsCalendarService monday,
+      GtfsCalendarService tuesday, GtfsCalendarService wednesday, GtfsCalendarService thursday,
+      GtfsCalendarService friday, GtfsCalendarService saturday, GtfsCalendarService sunday) {
+    super(new ImmutableMap.Builder<String, Object>()
+              .put("serviceId", serviceId)
+              .put("monday", monday)
+              .put("tuesday", tuesday)
+              .put("wednesday", wednesday)
+              .put("thursday", thursday)
+              .put("friday", friday)
+              .put("saturday", saturday)
+              .put("sunday", sunday)
```

Comment:
Wouldn't all possible CalendarHasNoActiveDaysNotice have the same value for monday, tuesday, wednesday etc.?

Reply:
Yep, i have removed all that and added a csv row number instead.